src/screens/ProfileScreen/elements/BioAndDMSetting.tsx

```diff
@@ -76,68 +108,105 @@ const BioAndDMSetting: React.FC<BioAndDMSettingProps> = ({
+              style={{
+                display: 'flex',
+                flexDirection: 'row',
+                justifyContent: 'center',
+                alignItems: 'center'
+              }}>
+              <PencilIcon />
+              <Text
+                style={{
+                  fontSize: 14,
+                  fontWeight: '500',
+                  color: COLORS.white,
+                  marginLeft: 4
+                }}>
+                Edit
+              </Text>
+            </View>
+          </Pressable>
         </View>
-      </Pressable>
-
-      <TouchableOpacity onPress={toggleSwitchAnon} style={styles.toggleSwitchAnon}>
-        <ToggleSwitch
-          value={isAnonymity}
-          onValueChange={toggleSwitchAnon}
-          labelLeft="Allow anonymous messages?"
-          circleActiveColor={COLORS.signed_primary}
-          activeTextColor={COLORS.signed_primary}
+        <Divider
+          style={{
+            marginVertical: 12
+          }}
         />
-      </TouchableOpacity>
-
-      {isAnonymity && (
-        <TouchableOpacity
-          onPress={toggleSwitchAnonAllowFollowing}
-          style={styles.toggleSwitchAnonFollowing}>
-          <ToggleSwitch
-            value={isAllowFollowingSendDM}
-            onValueChange={toggleSwitchAnonAllowFollowing}
-            labelLeft="Only allow anon DMs from users you follow?"
-            circleActiveColor={COLORS.signed_primary}
-            activeTextColor={COLORS.signed_primary}
-          />
+        <Text style={styles.bioText}>{addDotAndRemoveNewline(bio || '')}</Text>
+      </View>
+      <View
+        style={{
+          backgroundColor: COLORS.white,
+          borderRadius: 12,
+          padding: 12,
+          marginHorizontal: 12,
+          shadowColor: 'rgba(0, 0, 0, 0.04)',
+          shadowOffset: {
+            width: 0,
+            height: 1
+          },
+          shadowOpacity: 1,
+          shadowRadius: 2,
+          elevation: 2,
+          marginTop: -50,
+          borderWidth: 1,
+          borderColor: COLORS.gray
+        }}>
+        <Text
+          style={{
+            fontSize: 12,
+            fontWeight: '400',
+            color: COLORS.gray500,
+            marginBottom: 12
+          }}>
+          Other users will be able to reply to your prompt and direct message you.
+        </Text>
+        <TouchableOpacity onPress={toggleSwitchAnon}>
+          <CheckBoxCustom value={isAnonymity} label="Allow anonymous messages?" />
         </TouchableOpacity>
-      )}
+
+        {isAnonymity && (
+          <>
+            <Divider style={{marginVertical: 12, backgroundColor: COLORS.gray}} />
+            <TouchableOpacity onPress={toggleSwitchAnonAllowFollowing}>
+              <CheckBoxCustom
+                value={isAllowFollowingSendDM}
+                label="Only allow anon DMs from users you follow?"
+              />
+            </TouchableOpacity>
+          </>
+        )}
+      </View>
     </View>
   );
 };
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

This section contains significant changes to the layout and styling of the `BioAndDMSetting` component. The changes seem to be part of a UI redesign. However, there are several inline styles used throughout this section. It would be better to move these styles into the `StyleSheet.create` method at the bottom of the file for better readability and maintainability.



<!-- This is an auto-generated comment by OSS CodeRabbit -->